src/scripts/check-health.sh

```diff
@@ -2,6 +2,8 @@
 user="$(id -u)"
 if [ "$user" -ne 0 ]; then export SUDO="sudo"; else export SUDO=""; fi
 if grep "Alpine" < /etc/issue > /dev/null 2>&1; then
```

Comment:
This line can be a bit cleaner with a couple of changes:

```suggestion
if grep "Alpine" /etc/issue &> /dev/null; then
```

Assuming `grep` and `&>` work the same way in ash as they do in bash.

Reply:
&> does not work in ash which sucks because your revision is way cleaner. The ash equivalent is 2&>1